tests/conftest.py

```diff
@@ -0,0 +1,103 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from pathlib import Path
+import sys
+
+from everett.manager import ConfigManager, ConfigDictEnv, ConfigOSEnv
+from falcon.request import Request
+from falcon.testing.helpers import create_environ
+from falcon.testing.client import TestClient
+import markus
+import pytest
+
+
+# Add repository root so we can import antenna.
+REPO_ROOT = Path(__file__).parent.parent
+sys.path.insert(0, str(REPO_ROOT))
+
+from antenna.app import get_app, setup_logging  # noqa: E402
+from antenna.app import reset_verify_funs  # noqa: E402
+import antenna.libmarkus  # noqa: E402
+
+
+def pytest_runtest_setup():
+    # Make sure we set up logging and metrics to sane default values.
+    setup_logging(logging_level="DEBUG", debug=True, host_id="", processname="antenna")
+    markus.configure([{"class": "markus.backends.logging.LoggingMetrics"}])
+    # prevent antenna from reconfiguring markus
+    antenna.libmarkus._IS_MARKUS_SETUP = True
```

Comment:
I believe I figured out the source of this PytestUnraisableExceptionWarning:

<details><summary>traceback</summary>

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/_pytest/runner.py", line 341, in from_call
    result: Optional[TResult] = func()
                                ^^^^^^
  File "/usr/local/lib/python3.11/site-packages/_pytest/runner.py", line 262, in <lambda>
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pluggy/_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pluggy/_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pluggy/_callers.py", line 130, in _multicall
    teardown[0].send(outcome)
  File "/usr/local/lib/python3.11/site-packages/_pytest/unraisableexception.py", line 83, in pytest_runtest_setup
    yield from unraisable_exception_runtest_hook()
  File "/usr/local/lib/python3.11/site-packages/_pytest/unraisableexception.py", line 78, in unraisable_exception_runtest_hook
    warnings.warn(pytest.PytestUnraisableExceptionWarning(msg))
pytest.PytestUnraisableExceptionWarning: Exception ignored in: <socket.socket fd=-1, family=2, type=2, proto=17>

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/functools.py", line 519, in decorating_function
    wrapper = _lru_cache_wrapper(user_function, maxsize, typed, _CacheInfo)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ResourceWarning: unclosed <socket.socket fd=12, family=2, type=2, proto=17, laddr=('172.19.0.7', 44008), raddr=('172.19.0.4', 8125)>
```

</details>

the socket used by markus to write to statsd wasn't being closed properly. solution: make sure antenna doesn't call `markus.configure` after it's called here.

Reply:
I don't understand why this would be kicking up a `ResourceWarning` now. That's puzzling. Having said that, this setup code is not great. Instead of calling `markus.configure` directly, I'd call `antenna.libmarkus.setup_metrics` which has the guard for calling it multiple times.

Reply:
i don't either, other than some side effect of putting a test in tests/external, which i have now reverted and tests pass, so i'll address this in a separate PR.

Reply:
bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1884215